docs/fastdds/dds_layer/subscriber/sampleInfo/sampleInfo.rst

```diff
@@ -3,3 +3,183 @@
 SampleInfo
 ==========
 
+When a sample is retrieved form the :ref:`dds_layer_subscriber_dataReader`, in addition to the sample data,
+a :ref:`api_pim_sampleinfo` instance is returned.
+This object contains additional information that complements the returned data value and helps on it interpretation.
+For example, if the :ref:`dds_layer_subscriber_sampleInfo_validdata` value is ``false``, the
+:ref:`dds_layer_subscriber_dataReader` is not informing the application about a new value in the data instance,
+but a change on its status, and the returned data value must be discarded.
+
+Please, refer to the section :ref:`dds_layer_subscriber_accessreceived` for more information regarding how received
+data can be accessed on the :ref:`dds_layer_subscriber_dataReader`.
+
+The following sections describe the data members of :ref:`api_pim_sampleinfo`
+and the meaning of each one in relation to the returned sample data.
+
+.. toctree::
```

Comment:
This is not doing the trick for me

Reply:
Neither does for me. That was an attempt to solve the problem that didn't work... and I forgot to remove it afterwards because it does literally NOTHING